api/.env

```diff
@@ -0,0 +1,6 @@
+# These environment variables are referneced from within api/docker-compose.yaml to
```

Comment:
We could use `.env` files in a number of places, for Cloud Functions, unit testing, etc. Does this one have to live at the top level like this, or could it go in a subdirectory just for docker?

Reply:
[Caveat: see my reply to CH's comment, and a soon-to-be-pushed change which removes this file altogether.]

Re: user of docker vs. docker user. I think they can be – but we have code elsewhere which tries to align the Docker user ID with the user ID who called the dev script.

Re: .env file location, it looks like that's possible by specifying an 'env_file' location in the docker-compose.yaml (https://docs.docker.com/compose/environment-variables/#the-env-file).